src/main/java/com/infamous/combat_next/util/CombatUtil.java

```diff
@@ -159,8 +162,8 @@ public static void attackEmpty(Player player) {
     
     public static void resetAttackStrengthTicker(Player player, boolean miss, boolean allowDefault){
         if(miss && MeleeCombatConfigs.getAttackMissReducedCooldown().get()){
-            LivingEntityAccessor accessor = (LivingEntityAccessor) player;
-            accessor.setAttackStrengthTicker(Math.max(0, (int) (player.getCurrentItemAttackStrengthDelay() - MeleeCombatConfigs.getAttackMissCooldownTicks().get())));
+            player.resetAttackStrengthTicker();
```

Comment:
Does the attack strength ticker still need to be reset here?

Reply:
yes